### DISCUSSION TITLE
Talk Error

### DISCUSSION THREAD

**Discussion by krauthaus** [nextcloud/all-in-one#756] [Questions] [answered]
I got some trouble with my backups which filled up alle the space my system got. I delted them and restored my system via borgbackup, everything seems to working fine; but my talk is kind of messed up. It wont work at all, when I try to create a session with someone it just fails. Here is the beginning part of my talk-log:

`[ERR] [plugins/janus_streaming.c:janus_streaming_init:2005] Can't add 'live' mountpoint, no such file '/usr/share/janus/streams/radio.alaw'...

[ERR] [plugins/janus_streaming.c:janus_streaming_init:2072] Can't add 'ondemand' mountpoint, no such file '/usr/share/janus/streams/music.mulaw'...

JANUS Streaming plugin initialized!

Loading plugin 'libjanus_sip.so'...

JANUS SIP plugin initialized!

Loading plugin 'libjanus_videoroom.so'...

JANUS VideoRoom plugin initialized!

Loading plugin 'libjanus_voicemail.so'...

JANUS VoiceMail plugin initialized!

Loading plugin 'libjanus_videocall.so'...

JANUS VideoCall plugin initialized!

Loading plugin 'libjanus_lua.so'...

[echotest.lua] Loading...

[echotest.lua] Loaded

[echotest.lua] Initializing...

[echotest.lua] Initialized

Janus Lua plugin initialized!

Transport plugins folder: /usr/lib/x86_64-linux-gnu/janus/transports

Loading transport plugin 'libjanus_mqtt.so'...

[ERR] [config.c:janus_config_parse:205] Error parsing config file at line 30: syntax error

[WARN] Couldn't find .jcfg configuration file (janus.transport.mqtt), trying .cfg

[ERR] [config.c:janus_config_parse:191]   -- Error reading configuration file 'janus.transport.mqtt.cfg'... error 2 (No such file or directory)

MQTT SSL support disabled

[WARN] MQTT support disabled for both Janus and Admin API, giving up

JANUS MQTT transport plugin destroyed!

[WARN] The 'janus.transport.mqtt' plugin could not be initialized

Loading transport plugin 'libjanus_nanomsg.so'...

[ERR] [transports/janus_nanomsg.c:janus_nanomsg_init:224] Error binding Janus API Nanomsg socket to address 'ipc:///tmp/janus.ipc': 98 (Address already in use)

[WARN] No Nanomsg server started, giving up...

[WARN] The 'janus.transport.nanomsg' plugin could not be initialized

Loading transport plugin 'libjanus_http.so'...

Error launching Janus (error code 1), check the logs for more details

2022-05-26 11:48:00.644739 I | client.go:282: Client from 84.115.230.150 has RTT of 20 ms (20.021998ms)

2022-05-26 11:48:00.646125 I | backend_client.go:199: Capabilities expired for https://xxx.org/ocs/v2.php/cloud/capabilities, updating

2022-05-26 11:48:10.646076 I | backend_client.go:273: Could not get capabilities for https://xxx.org/ocs/v2.php/apps/spreed/api/v3/signaling/backend: Get "https://xxx.org/ocs/v2.php/cloud/capabilities": context deadline exceeded

2022-05-26 11:48:10.649669 I | backend_client.go:355: Could not send request {"type":"auth","auth":{"version":"1.0","params":{"userid":"XXX","ticket":"A6qpAosNyfCI7FuU:1653565680:XXX:fedf7ae133f010d5015cd0aaf0c38823d9d2e034835d895239488524574f1100"}}} to https://xxx.org/ocs/v2.php/apps/spreed/api/v3/signaling/backend: Post "https://xxx.org/ocs/v2.php/apps/spreed/api/v3/signaling/backend": context deadline exceeded

2022-05-26 11:48:12.124904 I | client.go:282: Client from 84.115.230.150 has RTT of 19 ms (19.079183ms)`

It would be nice if someone of you could point me to the right direction!

**Comment by szaimen** [comment, marked as answer]
Is port 3478 open?

**Comment by krauthaus** [comment]
Yeah, i opened that port. It was already working, but since it was kind of a testballoon for our office I used a small nuc and missed the backupfiles which dumped the small harddrive fully. I managed to backup, and since we get a real server for nextcloud I will just try my luck on a fresh install/migrate. 

Sorry for the late reply; this is  a freetime project of mine and atm we got our main seasion in the job.
